model/models.py

```diff
@@ -122,9 +121,68 @@ def forward(self, sub, rel):
 		x				= self.hidden_drop2(x)
 		x				= self.bn2(x)
 		x				= F.relu(x)
-
 		x = torch.mm(x, all_ent.transpose(1,0))
 		x += self.bias.expand_as(x)
 
 		score = torch.sigmoid(x)
 		return score
+
+class CompGCN_ConvKD(CompGCNBase):
```

Comment:
Are you sure that it is called convKD ? The only score function that I found with a similar name was convKB

Reply:
no, it should be called convKB